tests/test_container.py

```diff
@@ -73,3 +74,17 @@ def test_scene_remove():
     """Test Scene.remove()."""
     scene = Scene()
     container_remove(scene, lambda: scene.mobjects)
+
```

Comment:
**Issue :** First of all, your tests are perfect. 
Nevertheless, I don't think they should live in `test_container.py`. Could you create a new file, `test_value_tracker.py`?